src/services/logger.ts

```diff
@@ -108,18 +108,21 @@ const options = {
     service: logName,
     ddsource: 'nodejs',
     intakeRegion: 'eu',
+    ddtags: `region:${region}`,
   },
 }
 
 const getTransports = () => {
   const transports = [new winstonTransports.Console(options.console)]
 
   if (environment === 'production' || environment === 'staging') {
-    if (logToCloudWatch) {
+    if (logToCloudWatch || logToDataDog) {
```

Comment:
Did I miss where this was added to .envs and .env example?

const logToDataDog = process.env.LOG_TO_DATADOG === 'true'

Reply:
was added to envs on a previous PR but not to example, will do the example one